multicast/src/main/kotlin/com/dropbox/flow/multicast/SharedFlowProducer.kt

```diff
@@ -57,15 +54,11 @@ internal class SharedFlowProducer<T>(
                 collectionJob = scope.launch {
                     try {
                         src.catch {
-                            channelManager.send(
-                                DispatchError(
-                                    it
-                                )
-                            )
+                            channelManagerInbox(ChannelManager.Message.Dispatch.Error(it))
                         }.collect {
                             val ack = CompletableDeferred<Unit>()
-                            channelManager.send(
-                                DispatchValue(
+                            channelManagerInbox(
```

Comment:
calling inbox ( name) as a function seems a bit weird to me.
(can be fixed in another CL to not break the CL chain)

Reply:
yeah, I wasn't super happy with that either, but I think it's just years of reading java talking. Any better suggestions?

Reply:
I like send as the function

Reply:
as in `chennelManagerSend`?

Reply:
As in ChannelManagerInbox.send